packages/gatsby-theme-learning/src/hooks/use-course-details.ts

```diff
@@ -39,7 +39,14 @@ export const useFetchCourseDetails = (
 
   const { data, error, isLoading } = useSWR(
     shouldFetchData ? apiEndpoint : null,
-    (url) => fetcherWithToken(url, getAuthToken, learnApiBaseUrl, 'GET')
+    (url) => fetcherWithToken(url, getAuthToken, learnApiBaseUrl, 'GET'),
+    {
+      revalidateOnReconnect: false,
+      revalidateIfStale: false,
+      revalidateOnMount: true,
+      dedupingInterval: 10000000,
```

Comment:
@nkuehn regarding a reasonable deduping interval. This is clearly too aggressive: 160 hours meaning that the data will never be automatically revalidated. Shall we look more into minutes? or other ranges? Just to have a rough idea